depends/packages/native_clang.mk

```diff
@@ -8,6 +8,9 @@ $(package)_sha256_hash_linux=9ef854b71949f825362a119bf2597f744836cb571131ae6b721
 $(package)_download_file_darwin=clang+llvm-$($(package)_version)-x86_64-apple-darwin.tar.xz
 $(package)_file_name_darwin=clang-llvm-$($(package)_version)-x86_64-apple-darwin.tar.xz
 $(package)_sha256_hash_darwin=94ebeb70f17b6384e052c47fef24a6d70d3d949ab27b6c83d4ab7b298278ad6f
+$(package)_download_file_freebsd=clang+llvm-$($(package)_version)-amd64-unknown-freebsd11.tar.xz
```

Comment:
You mentioned that `libc++abi` is not in this package. Is `libcxxrt` provided? If so, we should use that instead of whatever the system provides.

Reply:
No. Neither `libcxxrt` nor `libc++abi` are provided pre-compiled for FreeBSD by the LLVM folks. `libcxxrt` is compatible with `libc++` and present in the FreeBSD base system. The only way to not rely on the base system would be to download and compile `libc++` and `libc++abi` together from source.

I actually tried building them from source using the `depends/` framework, but it was a lot more work than I had anticipated. The approach in this PR represents far fewer changes/new code.

Reply:
Yeah, I also tried building libc++ from source originally, and encountered the same issue (hence falling back on "let Clang do it"). In that case I'm fine with falling back on the system for FreeBSD, but I'm still a little puzzled as to why the system doesn't have libc++abi.

Reply:
If I had to guess, I'd say it's related to [licensing](https://freebsdfoundation.org/blog/libcxxrt-c-runtime-now-available-under-bsd-license/).